user_docs/en/changes.t2t

```diff
@@ -6,10 +6,27 @@ What's New in NVDA
 
 = 2024.1 =

```

Comment:
Similar to 2019.3, Python update should be mentioned.

```suggestion
= 2024.1 =

This release includes a significant update to Python (3.11), which improves the performance and security of NVDA.

```

Reply:
I think the plan in future will be to upgrade python yearly to the previous years release. (e.g. 3.12 2025.1, 3.13 2026.1, etc). The difference between 3.x releases is the same as the 2019.3 2to3 jump. I'm not sure whether its worth mentioning in the release blurb as it doesn't affect most users.
Are we aware of any noticeable performance improvements that didn't come from WASAPI?

Reply:
```suggestion
= 2024.1 =
This release updates our usage of Python to 3.11, which improves the security (and performance?) of NVDA.
```

Reply:
> 3.12 2025.1

Depending on the timing of releases and dependency updates, I'd hope we can eventually sync up with Python's release cadence so we're elegible for bugfix updates if and when they become available.

Reply:
NV Access's current stance is we would rather avoid being on "cutting edge" python. While we don't have a hard and fast rule, I think we would reject upgrading to a python release newer than 6 months at a minimum. The current ~1 year is pretty ideal right now.

Reply:
I'd say mentioning performance benefits, when we are not aware about any, sounds like either trying to advertise the release, or implying that newer version of Python must be more performant, just because it is more recent.